linera-base/src/identifiers.rs

```diff
@@ -1026,15 +1026,15 @@ mod tests {
     fn chain_ids() {
```

Comment:
It might be helpful to add root chains 1 and 2 here as well: They are probably used in a few READMEs, so this serves as documentation where one can look up the hash. It might also simplify any future changes to how chain IDs are computed.

Reply:
Good idea. What I can see now is that we enumerate from `1` READMEs (i.e. `ChainId::root(0)` is `CHAINID_1`).